src/constants/email-message.ts

```diff
@@ -0,0 +1,74 @@
+export const mailText = [
```

Comment:
Actually, the user receives just one email, and in the body of the email they can read what happened v.g. in the last hour.
The body would contain something like:
```
jackson created the article Python
Melanie associated taxonomy Math with Trigonometry
```
@kulla could you please update the issue with an example of notification email in English?